src/useDebounce.ts

```diff
@@ -1,22 +1,27 @@
 import { useState, useCallback, useRef, useEffect } from 'react';
 import useDebouncedCallback from './useDebouncedCallback';
 
+function refEquality<T>(left: T, right: T): boolean {
```

Comment:
maybe it's better to call it `valueEquality`. Because we compare prev and next values

Reply:
You're right. I renamed it to `valueEquality`.